modules/change-config-options-after-deployment.adoc

```diff
@@ -26,4 +26,8 @@ $ helm upgrade -n stackrox \
 ====
 You can also specify configuration values using the `--set` or `--set-file` parameters.
 However, these options are not saved, and it requires you to manually specify all the options again whenever you make changes.
+* Some changes, such as enabling a new component like Scanner V4, require new certificates to be issued for the component. Therefore, you must provide a CA when making these changes.
+** If the CA was generated by the Helm chart during the initial installation, you must retrieve these auto-generated values from the cluster and provide them to the 'helm upgrade' command. A command for retrieving the auto-generated values is included in the post-installation notes of the `central-services` Helm chart.
+** If the CA was generated outside of the Helm chart and provided during the installation of the `central-services` chart, then you must perform that action again when using 'helm upgrade', for example, by using the '--reuse-values' flag with the `helm upgrade` command.
```

Comment:
Same comments here as for `modules/change-config-options-after-deployment-central-service.adoc` above. 

You also need a line between your first bullet and the text above it so that it knows to start a list (it's currently parsing it as all inline)
![image](https://github.com/openshift/openshift-docs/assets/61474374/ef8421e6-b73b-4492-ab92-3f4c437e19f3)
```suggestion
You can also specify configuration values using the `--set` or `--set-file` parameters.
However, these options are not saved, and it requires you to manually specify all the options again whenever you make changes.

* Some changes, such as enabling a new component like Scanner V4, require new certificates to be issued for the component. Therefore, you must provide a CA when making these changes.
** If the CA was generated by the Helm chart during the initial installation, you must retrieve these auto-generated values from the cluster and provide them to the 'helm upgrade' command. A command for retrieving the auto-generated values is included in the post-installation notes of the `central-services` Helm chart.
** If the CA was generated outside of the Helm chart and provided during the installation of the `central-services` chart, then you must perform that action again when using 'helm upgrade', for example, by using the '--reuse-values' flag with the `helm upgrade` command.
```